core/client/app/mixins/ed-editor-api.js

```diff
@@ -129,7 +129,7 @@ var EditorAPI = Ember.Mixin.create({
 
         $textarea.focus();
         // Tell the editor it has changed, as programmatic replacements won't trigger this automatically
-        this.sendAction('onChange');
+        this.set('value', $textarea.val());
```

Comment:
@jaswilli, @novaugust & @kevinansfield I wonder if one of you could please help shed some light on what the right way to do this is? I added `this.sendAction('onChange')` to try to ensure that programmatic changes to the textarea were picked up, but it doesn't seem to be working, at least in Firefox.

There was a conversation about this in slack here: https://ghost.slack.com/archives/ghost/p1437908472000844

Any help would be much appreciated.